action.yml

```diff
@@ -13,13 +13,17 @@ inputs:
   otp-version:
     description: Version range or exact version of Erlang/OTP to use
   install-hex:
-    description: Whether to install Hex
+    description: Whether to install Hex (with Mix)
     default: true
   install-rebar:
-    description: Whether to install Rebar
+    description: Whether to install Rebar (with Mix)
     default: true
   rebar3-version:
     description: Version range or exact version of rebar3 to use
+  version-type:
```

Comment:
just wanted to say a big thank you for _this_ option in particular (and the whole project in general.) :D